plugins/state_monitor.py

```diff
@@ -0,0 +1,119 @@
+"""
+Plugin to monitor state varaibles from the charge controller
+"""
+
+# Local imports
+import brokkr.pipeline.base
+import brokkr.pipeline.decode
+import subprocess
+import os.path
+
+# Provide the script that will be called to send the messages about the states
+SCRIPT = os.path.expanduser("~/dev/mjolnir-hamma/scripts/status_bot.py")
```

Comment:
Now that I think of it, its a moot point here but if you were to do this, instead of specifying the full absolute path, which is brittle and will break if you move/rename anything, move it to a different Pi, etc., you should simply specify the path relative to `Path(__file__).parent`, i.e. `Path(__file__).parent.parent / "scripts" / "status_bot.py"`.